Makefile

```diff
@@ -285,27 +285,63 @@ minimaltools:
 dependency_check:
 	./tools/dependency_check.sh
 
-GEN_BASE_DIR ?= ./go/vt/topo/k8stopo
+install_k8s-code-generator: tools.go go.mod
+	go install k8s.io/code-generator/cmd/deepcopy-gen
+	go install k8s.io/code-generator/cmd/client-gen
+	go install k8s.io/code-generator/cmd/lister-gen
+	go install k8s.io/code-generator/cmd/informer-gen
 
-client_go_gen:
+DEEPCOPY_GEN=$(GOBIN)/deepcopy-gen
+CLIENT_GEN=$(GOBIN)/client-gen
+LISTER_GEN=$(GOBIN)/lister-gen
+INFORMER_GEN=$(GOBIN)/informer-gen
+
+GEN_BASE_DIR ?= vitess.io/vitess/go/vt/topo/k8stopo
+
+client_go_gen: install_k8s-code-generator
 	echo $$(date): Regenerating client-go code
 	# Delete and re-generate the deepcopy types
-	find $(GEN_BASE_DIR)/apis/topo/v1beta1 -type f -name 'zz_generated*' -exec rm '{}' \;
-	deepcopy-gen -i $(GEN_BASE_DIR)/apis/topo/v1beta1 -O zz_generated.deepcopy -o ./ --bounding-dirs $(GEN_BASE_DIR)/apis --go-header-file $(GEN_BASE_DIR)/boilerplate.go.txt
+	find $(VTROOT)/go/vt/topo/k8stopo/apis/topo/v1beta1 -name "zz_generated.deepcopy.go" -delete
```

Comment:
Is the `-delete`  flag available on mac versions of `find`?

Reply:
It works for me on Big Sur

Reply:
Shows up in the `man` page for `find` on catalina as well.